src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonPatchOperationsArrayProvider.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+using Microsoft.AspNetCore.Mvc.Abstractions;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Microsoft.AspNetCore.Mvc.Formatters.Json
+{
+    public class JsonPatchOperationsArrayProvider : IApiDescriptionProvider
+    {
+        private readonly IModelMetadataProvider _modelMetadataProvider;
+
+        public JsonPatchOperationsArrayProvider(
+            IModelMetadataProvider modelMetadataProvider)
+        {
+            _modelMetadataProvider = modelMetadataProvider;
+        }
+
+        /// <inheritdoc />
+        public int Order
+        {
+            get { return -999; }
```

Comment:
I'm _guessing_ `OnProvidersExecuting()` needs to run just after the one in `DefaultApiDescriptionProvider`. Whatever the reason for the value, add a comment describing it.